src/envoy/mixer/http_filter.cc

```diff
@@ -101,18 +104,46 @@ void CreateAuthIssuers(
 
 }  // namespace
 
+/**
+ * All http mixer filter stats. @see stats_macros.h
+ */
+// clang-format off
+#define ALL_HTTP_MIXER_FILTER_STATS(COUNTER, GAUGE)                           \
+  COUNTER(total_check_calls)                                                  \
```

Comment:
We also want separate check and report latencies as histograms. The histograms should use the same bucketization as used by envoy upstream_XXX counters.
upstream_mixer_XXX stats mash checks and reports together.

Reply:
@JimmyCYJ please file a separate issue to add Check/Report latency stats.   Don't want to put too much change into one PR.

Reply:
I filed a another issue here. https://github.com/istio/mixerclient/issues/324
Thanks.